ios/FirebaseContinue/FirebaseContinue/Classes/FCNContinue.h

```diff
@@ -18,32 +18,58 @@
+ * both an iOS device and macOS computer that are Apple Handoff enabled).
  * For more details, see: https://github.com/firebase/firebase-continue
+ *
+ * Please see the usage instructions in the relevant README file(s).
+ * There is also more specific documentation within the library itself below.
+ *
+ * TODO: Add unit tests, including tests while the app/Firebase is offline.
  */
 @interface FCNContinue : NSObject
 
 /**
- * Broadcasts to all possible clients (such as a Chrome extension) that the user may wish to
- * "continue" the provided activity (via the URL) within the provided application. This uses
- * Firebase Authentication to ensure the activity data is user-specific, and the Firebase
- * Realtime Database to facilitate data storage and transfer to all possible clients which could
- * allow the user to continue the activity.
+ * Attempts to asynchronously broadcast an Activity (codified as a URL) within an application
+ * that the currently signed in user may wish to continue elsewhere (in the immediate future)
+ * to all potential clients (ex. Chrome extension(s) and/or Apple Handoff enabled macOS computers)
+ * which could allow the user to do so by opening said URL.
  *
- * @param activityUrl The URL which, if visited by the user, would allow them to continue their
- * activity.
- * @param applicationName The name of the application the current user's activity is within.
- * This must match an application name listed in your Firebase Realtime Database rules for
- * Firebase Continue (as noted in the Firebase Continue setup guides).
- * @param completionBlock The optional block which is invoked when the broadcasting is complete.
- * The error in the block will be non-nil if the broadcast was unsuccessful.
+ * Note that, by design, only the most recently successfully broadcast Activity (for a given
+ * application) could possibly be relevant to the user. The Firebase Continue database rules
+ * and libraries enforce this. For more details, please see the relevant README file(s).
+ *
+ * @param activityUrl The URL which, if the current user were to navigate to, would allow the user
+ * to continue their Activity.
+ * @param applicationName The name of the application, as defined in the Firebase Realtime
+ * Database rules for Firebase Continue, that the user's Activity is within.
+ * @param completionBlock An optional completion block which is invoked on the main thread when the
+ * broadcast attempt is complete. The error provided to the block will be non-nil if the broadcast
+ * was unsuccessful.
  */
 + (void)broadcastToContinueActivityWithUrl:(NSString*)activityUrl
                  withinApplicationWithName:(NSString*)applicationName
-                                onComplete:(nullable void (^)(NSError* error))completionBlock;
+                       withCompletionBlock:(nullable FCNContinueCompletionBlock)completionBlock
+NS_SWIFT_NAME(broadcastToContinueActivity(withUrl:withinApplication:onComplete:));
```

Comment:
I was using Xcode's formatter and it seemed to want this as it is here - I'll indent it, though!